file-formats/ntta/type/zif_aff_ntta_v1.intf.abap

```diff
@@ -17,21 +17,21 @@ INTERFACE zif_aff_ntta_v1
       implementation_exit TYPE zif_aff_types_v1=>ty_object_name_30,
     END OF ty_general_information,
 
-    "! <p class="shorttext">Notetype</p>
-    "! Notetype
+    "! <p class="shorttext">Note Type</p>
+    "! Note Type
```

Comment:
Is "Note Type" planned to take up as SAPterm (written in upper case)? Currently I only find "node type" there.

Other object types like classes, tables, programs are usually written in lower case.

If "note type" is the right term, you should change the descriptions to 
```suggestion
    "! Note type
```

Same is valid for "Note type assignment"

Reply:
I changed it to lower case. We have no plans to add it to SAPterm.